### PR TITLE
Add feast recipe

### PR DESCRIPTION
**Pull request by basnijholt**

### CODE REVIEW

File: recipes/feast/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    - test -f "${PREFIX}/lib/libfeast_banded.a"
+    - test -f "${PREFIX}/lib/libfeast_dense.a"
+    - test -f "${PREFIX}/lib/libfeast_sparse.a"
+    - test -f "${PREFIX}/lib/libpfeast.a"
+    - test -f "${PREFIX}/lib/libpfeast_banded.a"
+    - test -f "${PREFIX}/lib/libpfeast_dense.a"
+    - test -f "${PREFIX}/lib/libpfeast_sparse.a"
+
+about:
+  home: http://www.feast-solver.org/
```
Comment (mariusvniekerk) [inline recipes/feast/meta.yaml]: remove the trailing slash as that does not resolve for some reason

File: recipes/feast/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
+{% set name = "feast" %}
+{% set version = "0.3.0" %}
```
Comment (mariusvniekerk) [inline recipes/feast/meta.yaml]: The version should be 3.0

File: recipes/feast/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://www.ecs.umass.edu/~polizzi/feast/m3-0/feast_3.0.tgz
+  sha256: {{ sha256 }}
+
+build:
+  number: 100
```
Comment (mariusvniekerk) [inline recipes/feast/meta.yaml]: Set this to 0 please

**Comment by marcelotrevisani** [comment]
ping @basnijholt

**Comment by basnijholt** [comment]
Thanks a lot for the review.

I was working on a [Python wrapper for FEAST](https://github.com/basnijholt/pyfeast) but then found that FEAST is already included in MKL and is much faster than with OpenBLAS.

The development of the wrapping has also come to a standstill for now.

Do you think it's still useful to merge this even if there is no defined purpose ATM?

**Comment by basnijholt** [comment]
I'll close this one since I do not foresee myself working on this in the (near term) future.

**Pull request closed without merging**